src/nemo_evaluator/solvers/openclaw.py

```
@@ -517,109 +404,183 @@ async def _solve_sandbox(self, task: SeedResult, sandbox: Sandbox) -> SolveResul
+                        raw_sessions.append(raw_session_jsonl)
+                    await self._cleanup_sessions_sandbox(sandbox)
+                active_session_id = f"nel-{task_id}-{session_spec['id']}-{uuid.uuid4().hex[:8]}"
+                session_ids.append(active_session_id)
+
+            effective_prompt = self._build_prompt(
+                str(session_spec["prompt"]),
+                _CONTAINER_WORKSPACE if workspace_path else None,
+            )
+            effective_prompts.append(effective_prompt)
+
+            prompt_file = f"/tmp/nel_prompt_{idx}.txt"
```

Ruff (0.15.14)

[error] Probable insecure usage of temporary file or directory: "/tmp/nel_prompt_"

(S108)